.appveyor-disabled.yml

```diff
@@ -0,0 +1,25 @@
+environment:
+  global:
+    CIBW_TEST_REQUIRES: pytest requests avro
+    CIBW_TEST_COMMAND: "pytest --import-mode=append --ignore={project}\\tests\\avro {project}\\tests"
+    # SDK v7.0 MSVC Express 2008's SetEnv.cmd script will fail if the
+    # /E:ON and /V:ON options are not enabled in the batch script intepreter
+    # See: http://stackoverflow.com/a/13751649/163740
+    CMD_IN_ENV: "cmd /E:ON /V:ON /C .\\tools\\appveyor\\run_with_env.cmd"
+
+#install:
+
+build_script:
+  - dir c:\python27
+  - dir c:\python27\include
+  - "curl https://raw.githubusercontent.com/chemeris/msinttypes/master/inttypes.h -o c:\\python27\\include\\inttypes.h"
```

Comment:
I know this is disabled, but should we really be referring to a branch rather than a specific gitref? Won't be reproducible if that file is ever changed on master.

Reply:
This is in no way complete, but serves a starting point for this future endeavour.
But your point is good, for that future PR :)